test/org/traccar/protocol/Jt600ProtocolDecoderTest.java

```diff
@@ -18,13 +18,13 @@ public void testDecode() throws Exception {
                 "2475605035891613002328091601152806086750106533350c00000000000a000000000000e1ff4f97007f1607"));
 
         verifyPosition(decoder, buffer(
-                "(3301210003,U01,040812,185302,T,22.564025,N,113.242329,E,5.21,152,9,32%,00000000000011,10133,5173,22,100,1)"));
+                "(3301210003,U01,040812,185302,T,22.564025,N,113.242329,E,5.21,152,9,32%,00000000000011,10133,5173,22,100,1,18)"));
```

Comment:
From my understanding current protocol implementation **always** sends check sum for U01, U02, U03 messages types. Are you absolutely sure that these are valid messages examples?

I have updated these tests because I was curious that the implementation supporting Uxx types of messages was only added recently (for example it does not exist in 3.7 for sure) and for the same guy, for whom I am working on these protocol updates. Thus, according to the documentation and to the week of testing, the correct message format is used in tests right now.